components/user-button.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import { SignOutButton, useUser } from "@clerk/clerk-react";
+
+export const UserButton = () => {
+  const user = useUser();
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger>
+        <Avatar>
+          <AvatarImage src={user.user?.imageUrl} alt="@shadcn" />
```

Comment:
The `alt` attribute is what is read by a screen reader for visually impaired users. Let's pass in the users name here

```tsx
alt={user.user?.name}
```

Reply:
Also might be easier to just destructure `user`

```tsx
const { user } = useUser();
```